baseapp/baseapp.go

```diff
@@ -488,7 +488,10 @@ func (app *BaseApp) getContextForTx(mode runTxMode, txBytes []byte) context.Cont
 		WithTxBytes(txBytes).
 		WithVoteInfos(app.voteInfos)
 
-	ctx = ctx.WithConsensusParams(app.GetConsensusParams(ctx))
+	if mode != runTxModeDeliver {
```

Comment:
Why was this modified? It should be harmless to always have this populated.

Reply:
yeah, sounds fine to me

Reply:
A little bit more efficient? since the BeginBlock already set it for the consensus connection.

Reply:
I don't think the change is worth it. There's no real "cost" here.

Reply:
But I don't mean to block here. Just a suggestion 👍

Reply:
done